app/views/course/assessment/submission/submissions/_submission.html.slim

```diff
@@ -0,0 +1,9 @@
+= content_tag_for(:tr, submission) do
+  td
+    = link_to edit_course_assessment_submission_path(current_course, submission.assessment,
+                                                     submission) do
+      = format_inline_text(submission.course_user.name)
+  td = submission.workflow_state.capitalize
```

Comment:
I'll do this in another PR if that is okay? Will go and find the places where this is used (I think there is another place where this is used also). This is related to #984